packages/scrubber-bar/src/scrubber-bar.ts

```diff
@@ -0,0 +1,185 @@
+import {
+  LitElement,
+  html,
+  css,
+  customElement,
+  property,
+  PropertyValues,
+  TemplateResult,
+  CSSResult,
+} from 'lit-element';
+
+@customElement('scrubber-bar')
+export default class ScrubberBar extends LitElement {
+  @property({ type: Number }) percentComplete = 0;
+
+  userInteracting = false;
+
+  render(): TemplateResult {
+    return html`
+      <div class="container">
+        <input
+          id="slider"
+          type="range"
+          min="0"
+          max="100"
```

Comment:
Question: how do we change the `min`, `max` values, if we want to use another range like 1-5?

Reply:
So right now it's based off of a percentage, but I guess I could just pass around the raw value.

Reply:
or make that the default if no values are passed

Reply:
I re-jiggered this altogether and made it a value since that's the lowest common denominator. The value is a percentage if you set the range 0-100 so it works out good :)